vulnerabilities/importers/apache_httpd.py

```diff
@@ -40,56 +49,121 @@ class ApacheHTTPDDataSourceConfiguration(DataSourceConfiguration):
 class ApacheHTTPDDataSource(DataSource):
 
     CONFIG_CLASS = ApacheHTTPDDataSourceConfiguration
-    url = "https://httpd.apache.org/security/vulnerabilities-httpd.xml"
+    base_url = "https://httpd.apache.org/security/json/"
+
+    def set_api(self):
+        self.version_api = GitHubTagsAPI()
+        asyncio.run(self.version_api.load_api(["apache/httpd"]))
 
     def updated_advisories(self):
         # Etags are like hashes of web responses. We maintain
         # (url, etag) mappings in the DB. `create_etag`  creates
         # (url, etag) pair. If a (url, etag) already exists then the code
         # skips processing the response further to avoid duplicate work
 
-        if create_etag(data_src=self, url=self.url, etag_key="ETag"):
-            data = fetch_xml(self.url)
-            advisories = to_advisories(data)
+        if create_etag(data_src=self, url=self.base_url, etag_key="ETag"):
```

Comment:
This isn't valid now because the provided endpoint doesn't provide etags in the headers. Remove this if statement and the emtpy list return below

Reply:
Also the comments related to  docs